front/temporal/upsert_tables/activities.ts

```diff
@@ -0,0 +1,138 @@
+import { Storage } from "@google-cloud/storage";
+import { isLeft } from "fp-ts/lib/Either";
+import * as reporter from "io-ts-reporters";
+
+import { upsertTableFromCsv } from "@app/lib/api/tables";
+import { Authenticator } from "@app/lib/auth";
+import { DataSourceResource } from "@app/lib/resources/data_source_resource";
+import type { WorkflowError } from "@app/lib/temporal_monitoring";
+import { EnqueueUpsertTable } from "@app/lib/upsert_queue";
+import mainLogger from "@app/logger/logger";
+import { statsDClient } from "@app/logger/withlogging";
+import config from "@app/temporal/config";
+
+export async function upsertTableActivity(
```

Comment:
Duplicated from the existing one, we can't use the same as Temporal only bundles what exist in the folder.

Reply:
:+1: fine since we'll remove the initial one soon right?

Reply:
Yep